browser/resources/settings/default_brave_shields_page/brave_adblock_browser_proxy.ts

```diff
@@ -23,92 +43,75 @@ export enum ErrorCode {
-  updateFilterList(uuid: string): Promise<boolean>
-  getListSubscriptions(): Promise<any> // TODO(petemill): Define the expected type
-  getCustomFilters(): Promise<any> // TODO(petemill): Define the expected type
-  setSubscriptionEnabled(url: string, enabled: boolean)
-  addSubscription(url: string)
-  addWebUiListener(eventName: string, callback: Function)
-  updateSubscription(url: string)
-  deleteSubscription(url: string)
-  viewSubscription(url: string)
-  getCustomScriptlets(): Promise<Scriptlet[]>
-  addCustomScriptlet(scriptlet: Scriptlet): Promise<ErrorCode>
-  updateCustomScriptlet(name: string, scriptlet: Scriptlet): Promise<ErrorCode>
-  removeCustomScriptlet(name: string): Promise<ErrorCode>
+  getRegionalLists: () => Promise<FilterList[]>
+  enableFilterList: (uuid: string, enabled: boolean) => void
+  updateFilterLists: () => Promise<boolean>
+  getCustomFilters: () => Promise<string>
+  updateCustomFilters: (value: string) => void
+  getListSubscriptions: () => Promise<any>
+  setSubscriptionEnabled: (url: string, enabled: boolean) => void
+  addSubscription: (url: string) => void
+  updateSubscription: (url: string) => void
+  deleteSubscription: (url: string) => void
+  viewSubscription: (url: string) => void
+  getCustomScriptlets: () => Promise<Scriptlet[]>
+  addCustomScriptlet: (scriptlet: Scriptlet) => Promise<ErrorCode>
+  updateCustomScriptlet: (name: string, scriptlet: Scriptlet) =>
+    Promise<ErrorCode>
+  removeCustomScriptlet: (name: string) => Promise<ErrorCode>
+  addWebUiListener: (eventName: string, callback: Function) => void
 }
 
 export class BraveAdblockBrowserProxyImpl implements BraveAdblockBrowserProxy {
   static getInstance() {
-    return instance || (instance = new BraveAdblockBrowserProxyImpl());
+    return instance || (instance = new BraveAdblockBrowserProxyImpl())
   }
 
-  /** @returns {Promise} */
-  getRegionalLists () {
+  getRegionalLists (): Promise<FilterList[]> {
     return sendWithPromise('brave_adblock.getRegionalLists')
   }
 
-  /** @returns {Promise} */
-  getListSubscriptions () {
-    return sendWithPromise('brave_adblock.getListSubscriptions')
+  enableFilterList (uuid: string, enabled: boolean): void {
+    chrome.send('brave_adblock.enableFilterList', [uuid, enabled])
+  }
+
+  updateFilterLists (): Promise<boolean> {
+    return sendWithPromise('brave_adblock.updateFilterLists')
   }
 
-  /** @returns {Promise} */
-  getCustomFilters () {
+  getCustomFilters (): Promise<string> {
     return sendWithPromise('brave_adblock.getCustomFilters')
   }
 
-  enableFilterList (uuid, enabled) {
-    chrome.send('brave_adblock.enableFilterList', [uuid, enabled])
+  updateCustomFilters (value: string): void {
+    chrome.send('brave_adblock.updateCustomFilters', [value])
   }
 
-  /** @returns {Promise<boolean>} */
-  updateFilterLists () {
-    return sendWithPromise('brave_adblock.updateFilterLists')
+  getListSubscriptions (): Promise<any> {
```

Comment:
```suggestion
  getListSubscriptions () {
```

Reply:
Sorry, think I might've implied the wrong thing - you should keep the `Promise<boolean>` or `Promise<string>` but remove the `Promise<any>` (because that's what `sendWithPromise` returns).

Reply:
Ahh, gotcha. Will fix that up when I squash everything, thank you!

Reply:
After speaking with @fallaciousreasoning we decided this is fine (and probably preferred), since return type is already specified in interface and can just be inferred.